File: 217-ContainsDuplicate/217-ContainsDuplicate.py
# Last updated: 4/3/2026, 2:01:19 PM
class Solution(object):
    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        nums.sort()
        for i in range(1,len(nums)):
            if nums[i]==nums[i-1]:
                return True

        return False

File: 217-ContainsDuplicate/217-ContainsDuplicate.cpp
// Last updated: 4/3/2026, 2:01:16 PM
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        unordered_set<int> st;
        for(int num : nums)
            st.insert(num);
        return st.size() != nums.size();
    }
};